packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -2093,6 +2097,38 @@ class IrisGrid extends Component<IrisGridProps, IrisGridState> {
     });
   }
 
+  async setPartitionConfig(
```

Comment:
Name this `selectPartitionKeyFromTable`, and add JSDocs as well.
`set` implies you're passing in the config to set, whereas `select` is more accurate in what we're doing here (selecting the row).
Also don't need to pass the `model` in... `IrisGrid` already has the model.